tools/cythonize.py

```diff
@@ -200,6 +200,14 @@ def process(path, fromfile, tofile, processor_function, hash_db, pxi_hashes, loc
         # store hash in db
         hash_db[normpath(fullfrompath)] = current_hash
 
+def process_generate_pyx(path, lock):
+    with lock:
+        print('Running {}'.format(path))
+    ret = subprocess.call([sys.executable, path])
+    with lock:
+        if ret != 0:
```

Comment:
Probably doesn't matter either way. The lock is mainly to synchronize stuff that prints and modifications of the hash db, so that you don't get mixed-up output.